cfme/common/provider.py

```diff
@@ -744,7 +744,7 @@ def refresh_provider_relationships_ui(self, from_list_view=False, wait=0, delay=
         if from_list_view:
             view = navigate_to(self, 'All')
             entity = view.entities.get_entity(name=self.name, surf_pages=True)
-            entity.check()
+            entity.ensure_checked()
```

Comment:
I haven't found a usage of this. It is the normal implementation of ensure_checked() that has been tested elsewhere.